src/components/common/ActionChip.vue

```diff
@@ -1,21 +1,22 @@
 <template>
   <div>
     <v-chip v-if="actionableItem.action === ActionTypes.ADDED"
-            x-small label color="#1669BB" text-color="white">ADDED</v-chip>
+            x-small label color="primary" text-color="white">ADDED</v-chip>
     <v-chip v-if="actionableItem.action === ActionTypes.EDITED"
-            x-small label color="#1669BB" text-color="white">CORRECTED</v-chip>
+            x-small label color="primary" text-color="white">{{editedLabel}}</v-chip>
     <v-chip v-if="actionableItem.action === ActionTypes.REMOVED"
-            x-small label color="#E0E0E0" text-color="grey darken-1">REMOVED</v-chip>
+            x-small label color="#E0E0E0" text-color="#212529">REMOVED</v-chip>
```

Comment:
Should the v-chip in ListPeopleAndRoles be updated like this one too?

Reply:
You bet! I have a commit for the Shared Repo coming to update some things.
After that, i will pull in the latest version and remove this from the local repo and use the shared component throughout.